app/cdap/components/WrangleHome/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+import React from 'react';
+import OngoingDataExploration from './Components/OngoingDataExploration';
+import WrangleCard from './Components/WrangleCard';
+import WrangleHomeTitle from './Components/WrangleHomeTitle';
+import { GradientLine } from './icons';
+import { HeaderImage } from './icons';
+import { useStyles } from './styles';
+
+const WranglerHomeNew = () => {
+  const classes = useStyles();
+  return (
+    <Box className={classes.wrapper} data-testid="wrangler-home-new-parent">
+      <Box className={classes.subHeader}>
+        <Typography className={classes.welcome}>
+          Hi David, <br />
```

Comment:
Some configurations of CDAP don't have authentication, so there's no user name.

Reply:
updated the welcome message to Hello and removed any mention of name